### PR TITLE
Add missing permissions to kafka-instances script

### PR DESCRIPTION
**Pull request by k-wall**
statefulset and pod permissions were absent, this caused the commands that query kafka state to fail.

### CODE REVIEW

**Comment by georgettica** [comment]
nod to last time the checks were made
https://github.com/openshift/managed-scripts/pull/41

what was missing there that you caught now?

**Comment by k-wall** [comment]
yes, indeed, sorry for the churn on this.

For this mistake to have slipped through there must have been some mistake made with the application of the testing procedure when rachel did the original work.    She was using https://source.redhat.com/groups/public/openshiftplatformsre/wiki/backplane_scripts    "How to test the script before creating the PR." .  I'm unsure why she didn't see these errors. 
 
In addition we are still having problems running `ocm backplane testjobs` which we are still working through separately. (@StevenTobin )

File: scripts/CEE/get-kafka-instance-state/metadata.yaml [hunk 1]
```diff
@@ -41,6 +41,19 @@ rbac:
...
           resources:
             - "pods/exec"
+        - verbs:
+            - "get"
+            - "list"
+          apiGroups:
+            - "apps"
+          resources:
+            - "statefulsets"
+        - verbs:
```
Comment (georgettica) [inline scripts/CEE/get-kafka-instance-state/metadata.yaml:51]: Isn't that redundant / too permissive?
Why do you need this section?

**Comment by georgettica** [comment]
Hopefully the last 

But I think I found a permission issue I notes
After that I'll lgtm

File: scripts/CEE/get-kafka-instance-state/metadata.yaml (hunk 1 above)
Comment (k-wall) [reply scripts/CEE/get-kafka-instance-state/metadata.yaml:51]: thanks, well spotted.  I made a mistake copying back from the rbac yaml test case into the metadata.yml.

**Pull request merged**